app/tests/integration/commands.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Test command constants."""
+
+import dataclasses
+
+
+@dataclasses.dataclass
+class Commands:
+    """Test commands to execute.
+
+    Attributes:
+        name: The test name.
+        command: The command to execute.
+        env: Additional run envs.
+        external: OpenStack VM test only (Chroot unsupported test).
+    """
+
+    name: str
+    command: str
+    env: dict | None = None
+    external: bool = False
+
+
+# Most of the commands are already tested in the github-runner-operator repository:
```

Comment:
the outcommented commands will be removed in a follow-up PR, together with the code to support microk8s&juju